projects/exui-common-lib/src/lib/components/share-case/share-case.component.spec.ts

```diff
@@ -45,4 +48,73 @@ describe('ShareCaseComponent', () => {
     fixture.detectChanges();
     expect(fixture.debugElement.nativeElement.querySelector('#no-case-display').textContent).toContain('No cases to display.');
   });
+
+  it('should disable continue button', () => {
+    sharedCases = [{
+      caseId: '9417373995765133',
+      caseTitle: 'Sam Green Vs Williams Lee',
+      sharedWith: [
+        {
+          idamId: 'u666666',
+          firstName: 'Kate',
+          lastName: 'Grant',
+          email: 'kate.grant@lambbrooks.com'
+        }]
+    }];
+    component.state$ = of(sharedCases);
+    fixture.detectChanges();
+    expect(component.isDisabledContinue()).toBeTruthy();
+  });
+
+  it('should enable continue button when remove user', () => {
+    sharedCases = [{
+      caseId: '9417373995765133',
+      caseTitle: 'Sam Green Vs Williams Lee',
+      sharedWith: [
+        {
+          idamId: 'u666666',
+          firstName: 'Kate',
+          lastName: 'Grant',
+          email: 'kate.grant@lambbrooks.com'
+        }],
+      pendingUnshares: [
+        {
+          idamId: 'u777777',
+          firstName: 'Nick',
+          lastName: 'Rodrigues',
+          email: 'nick.rodrigues@lambbrooks.com'
+        }]
+    }];
+    component.state$ = of(sharedCases);
+    fixture.detectChanges();
+    expect(component.isDisabledContinue()).toBeFalsy();
+  });
+
+  it('should enable continue button when added user', () => {
+    sharedCases = [{
+      caseId: '9417373995765133',
+      caseTitle: 'Sam Green Vs Williams Lee',
+      sharedWith: [
+        {
+          idamId: 'u666666',
+          firstName: 'Kate',
+          lastName: 'Grant',
+          email: 'kate.grant@lambbrooks.com'
+        }],
+      pendingShares: [
+        {
+          idamId: 'u888888',
+          firstName: 'Joel',
+          lastName: 'Molloy',
+          email: 'joel.molloy@lambbrooks.com'
+        }]
+    }];
+    component.state$ = of(sharedCases);
+    fixture.detectChanges();
+    expect(component.isDisabledContinue()).toBeFalsy();
+  });
+
```

Comment:
Can you please add a test continue button state when sharedCases size set to 0. Expecting it to be disabled when sharedCases size is 0.